hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -53,6 +53,24 @@
     <tag>OZONE, CONTAINER, MANAGEMENT</tag>
     <description>The ipc port number of container.</description>
   </property>
+  <property>
+    <name>dfs.container.ratis.datastream.ipc</name>
+    <value>9890</value>
```

Comment:
These values are different from the values in the java conf file.

Reply:
I will fix the problem and we should not use port 9890. This will conflict with the port in docker-compose